builtin/game/item_entity.lua

```diff
@@ -173,17 +176,22 @@ core.register_entity(":__builtin:item", {
 		local v = self.object:getvelocity()
 		if not core.registered_nodes[nn] or core.registered_nodes[nn].walkable and v.y == 0 then
 			if self.physical_state then
-				local own_stack = ItemStack(self.object:get_luaentity().itemstring)
-				-- Merge with close entities of the same item
-				for _, object in ipairs(core.get_objects_inside_radius(p, 0.8)) do
-					local obj = object:get_luaentity()
-					if obj and obj.name == "__builtin:item"
-							and obj.physical_state == false then
-						if self:try_merge_with(own_stack, object, obj) then
-							return
+				if self.mergeable ~= false then
```

Comment:
You need to write these kinds of things as comments in the code. This information is of no use here in the long run but would be very useful if it was in the code.